docs/user-guide/faq.md

```diff
@@ -78,6 +78,19 @@ You can also take advantage of `selector-*` rules to ban certain categories of s
 
 If you're using SUITCSS, you'll probably want to use [their shareable config](https://github.com/suitcss/stylelint-config-suitcss).
 
+## How do I configure the `*-pattern` rules for common CSS naming conventions like kebab-case?
+
+Use the regex that corresponds to your chosen convention:
+
+-   kebab-case: `^([a-z][a-z0-9]*)(-[a-z0-9]+)*$`
+-   lowerCamelCase: `^[a-z][a-zA-Z0-9]+$`
+-   snake\_case: `^([a-z][a-z0-9]*)(_[a-z0-9]+)*$`
+-   UpperCamelCase: `^[A-Z][a-zA-Z0-9]+$`
+
+e.g. for lowerCamelCase class selectors, use `"selector-class-pattern": "^[a-z][a-zA-Z0-9]+$"`.
+
+All these patterns disallow CSS identifiers that start with a digit, two hyphens, or a hyphen followed by a digit.
```

Comment:
I didn't add it as I think it's covered in the [previous question and answer](https://github.com/stylelint/stylelint/blob/ac154debbf628d589b9fb1bb7788eca80ae0bf00/docs/user-guide/faq.md#how-should-i-lint-my-css-that-follows-a-bem-like-methodology). However, I've now tweaked the previous question so it's more aligned with this new one. Better?